src/cbl-mariner/2.0/helix/amd64/Dockerfile

```
@@ -0,0 +1,39 @@
+FROM cblmariner.azurecr.io/base/core:2.0
```

```suggestion
FROM mcr.microsoft.com/cbl-mariner/base/core:2.0
```

I see the 1.0 stuff is still using cblmariner.azurecr.io. That's a deprecated location for Mariner images. It's still kept current but we should be using mcr instead.

---

updated. It seems like the base 2.0 image already changed that.